ts/features/bonus/bpd/components/superCashbackRanking/LastPositionItem.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+import { formatNumberWithNoDigits } from "../../../../../utils/stringBuilder";
+import I18n from "../../../../../i18n";
+import RankPositionItem from "./RankPositionItem";
+
+type Props = {
+  transactionsNumber: number;
+  superCashbackAmount: number;
+  lastAvailablePosition: number;
+};
+
+export const LastPositionItem: React.FunctionComponent<Props> = (
+  props: Props
+) => (
+  <RankPositionItem
+    transactionsNumber={props.transactionsNumber}
+    superCashbackAmount={props.superCashbackAmount}
+    boxedLabel={"100k"}
```

Comment:
Have we a way to avoid this hardcoded value?
Can we extract somewhere (from some API) the last rank position?

Reply:
Maybe an idea could be the solution developed in 6123c7e what do you think?